src/core.js

```diff
@@ -2440,10 +2435,12 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
    * @memberof Core#
    * @function alter
    * @param {string} action Possible alter operations:
-   *  * `'insert_row'`
-   *  * `'insert_col'`
-   *  * `'remove_row'`
-   *  * `'remove_col'`.
+   *  <ul>
+   *    <li> `'insert_row'` </li>
+   *    <li> `'insert_col'` </li>
+   *    <li> `'remove_row'` </li>
+   *    <li> `'remove_col'` </li>
+   * </ul>.
```

Comment:
HTML here? We can't use markdown in JSDocs? That's a regression

Reply:
It is a param description that goes to the table as a one-line string. Using `*` does not generate a list.

Reply:
It does for the current docs:

![Screenshot 2021-06-28 at 13 08 37](https://user-images.githubusercontent.com/4718998/123627163-f6b47f80-d811-11eb-8eff-0fb659fbe2f9.png)

So is there a regressions with markdown in JSDoc?

Reply:
Currently, we use HTML under the hood. In the new docs, all is powered by MD. 

In MD is no syntax for adding a list inside a table. It requires HTML. 
Extending the jsdoc-to-markdown (exactly `dmd` package - a part of `jsdoc-to-markdown`) is quite hard and unprofitable.

To be honest, I would like to replace the `dmd` by own implementation, however, it needs time and a lot of work. As we have some more important topics on the hood, I suppose it can't be realized now, and in the future too. The more that we migrate to TS, and finally we are going to use **ts**docs instead of **js**docs.

Reply:
True. Some MD implementations does not support that.

As long as we're aware of new limitation in syntax it's ok. We can resolve this